### PR TITLE
ReadMe

### PR DESCRIPTION
**Pull request by JBrabson**
Pending:

- View Demo/Report Bug/Request Feature links: need formatted with url OR deleted, if unwanted
- ABOUT THE PROJECT section: needs app or app title page screen shot
- USAGE section: not completed/Documentation link to be deleted if unwanted
- Proofread por favor
- 
## Checklist
- [n/a] Tests written where required
- [n/a] All tests passing
- [n/a] SimpleCov at 100%
- [n/a RuboCop complaints resolved

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,123 @@
...
+See the open [issues](https://github.com/Caleb1991/epeolatry_front_end/issues) for a list of proposed features (and known issues).
+</a>
+
+
+<a id="contribute">
+<!-- CONTRIBUTE -->
+  
+## Contribute | <em>impart</em>
+
+Did our shmorgishborg of sayings, sentences, synonyms and sesquipedalians inspire a great idea for add-ons? Did you have that, "I'd really like to see THIS" moment while using the app? Feel free to share!
```
Comment (aliyamerali) [inline README.md:105]: *Love* this invitation to collaborate!

File: README.md [hunk 2]
```diff
@@ -1,24 +1,123 @@
...
+  
+## Getting Started | <em>to commence</em>
+
+Commence Cranium and Concordance Expansion Now... on [HEROKU](https://epeolatrys-rex.herokuapp.com)
+</a>
+
+<a id="usage">
+<!-- USAGE -->
+  
+## Usage | <em>utilization</em>
```
Comment (aliyamerali) [inline README.md:84]: Do you think we need this info for the frontend? Or should we just direct folks to Heroku and link to the BE if people want more info?

File: README.md [hunk 3]
```diff
@@ -1,24 +1,123 @@
...
-* System dependencies
+  <h3 align="center">e . p . e . o . l . a . t . r . y</h3>
 
-* Configuration
+  <p align="center">
+    an app to catalog your literary and vernacular journey. <br> propelling your pronunciation prowess of perplexing prose <br> <em>epi...olive...uh...tree?</em> --> <strong>epeolatry</strong> <br>
+    <br />
+    <a href="https://epeolatrys-rex.herokuapp.com/">View Demo</a>
+    ·
+    <a href="BUGSREPORTLINKHERE">Report Bug</a>
```
Comment (aliyamerali) [inline README.md:23]: Is there a way to link to creating a github issue? I feel like that would maybe be the best place to direct both bug report + feature requests

Comment (aliyamerali) [reply README.md:23]: I wonder if this will work for non collaborators? 

https://github.com/Caleb1991/epeolatry_front_end/issues/new

**Review by aliyamerali — APPROVED** [review]
Looking great! I expanded my vocabulary just by reading this 😛

**Review by Aferencz1987 — APPROVED** [review]

**Pull request merged**